daemon/api.go

```diff
@@ -327,6 +332,14 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 		}
 	}
 
+	if len(includeTypes) > 0 {
```

Comment:
the store actually supports filtering by type, maybe we want to do that?
E.g. `https://search.apps.ubuntu.com/api/v1/search?q=content%3A%22gadget%22`
Not sure whether it supports searching multiple simultaneously, I'd have to ask.